LibertyDSNP/example-client
# prevent torus from erroring when used for login

Purpose
---------------
Initial support for the SDK only worked if you logged in through Metamask. Torus logins fail, and if you had already logged in with Torus the app would not even load.


Solution
---------------
This PR adds the ability to choose the correct provider based on which login method had been selected.


Change summary
---------------
* A detailed list of bulleted
* changes that go into detail about
* the specifics of the changes
* to the codebase


Steps to Verify
----------------
1. Open the example client.
2. Sign up for Torus
3. Log in through Torus (you should get no error)
4. Attempt to make a Post (you should get a Torus popup to sign the batch announcement).
